src/content/docs/en/guides/imports.mdx

```diff
@@ -188,9 +188,6 @@ import logoUrl from '@assets/logo.png?url';
 ---
 ```
 
-These aliases are also integrated automatically into [VS Code](https://code.visualstudio.com/docs/languages/jsconfig) and other editors.
```

Comment:
I think this line is OK to keep! It doesn't mention `jsconfig` by name, even though it goes to page about it. And that page says that `jsconfig` is a descendent of `tsconfig`, so it shouldn't be confusing if people go read it.

Let's keep it in, because this line tells people that the aliases they set will be used in their editor!

Reply:
Ok